lib/util/http/index.ts

```diff
@@ -0,0 +1,118 @@
+import is from '@sindresorhus/is/dist';
+import URL from 'url';
+import got from '../got';
+
+interface OutgoingHttpHeaders {
+  [header: string]: number | string | string[] | undefined;
+}
+
+export interface HttpOptions {
+  auth?: string;
```

Comment:
Used by packagist only and passed transparently by `got` through to Node.js's `http`. Could possibly be removed and left for our got authentication layer.

Per Node.js docs:
> `auth` `<string>` Basic authentication i.e. `'user:password'` to compute an Authorization header.

Reply:
Unsupported in got v10

Reply:
It doesn't pass the `auth` value through to `http`?

Reply:
Per https://github.com/sindresorhus/got/blob/982275e184e994398d85eb61b036f10988e03943/readme.md#options

options.auth has been replaced with options.username & options.password.